scripts/update_material_icon_font_and_names.py

```diff
@@ -75,3 +83,33 @@
 
 with open(NAMES_MODULE_PATH, "w") as file:
     file.write(updated_script_content)
+
+# Fetch the content from the URL
+# We use custom User-Agent header here to get .woff2 font instead of .ttf
+response = requests.get(
+    MATERIAL_ICONS_FONT_URL,
+    headers={
+        "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 14_7_1) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.4.1 Safari/605.1.15"
+    },
+)
+
+
+# Search for the font file URL in the response
+match = re.search(FONT_FILE_URL_REGEX, response.text)
+
+font_url = None
+
+if match:
+    font_url = match.group(1)
+    print("Extracted URL:", font_url)
+else:
+    print("URL not found")
```

Comment:
**suggestion (non-blocking):** Would it make sense to exit the process with code `1` to indicate failure?